api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,486 @@
+swagger: '2.0'
+info:
+  title: Nakadi Event Bus API Definition
+  description: |
+    -------------------------------
+    Nakadi Event Bus API Definition
+    -------------------------------
+
+    * Low-level API
+    * Stantardised event format (see #/definitions/Event)
+    * Partition strategy: event store will infer the partition from the `partitioning_key` in the message body
+
+  version: '0.4'
+  contact:
+    name: Team Aruha @ Zalando
+    email: nakadi-users@zalando.de
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  oauth2:
+    type: oauth2
+    flow: implicit
+    authorizationUrl: 'https://auth.example.com/oauth2/tokeninfo'
+
+paths:
+  /metrics:
+    get:
+      tags:
+        - monitoring
+      summary: Get monitoring metrics
+      responses:
+        '200':
+          description: Metrics data
+          schema:
+            $ref: '#/definitions/Metrics'
+        '401':
+          description: User is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+  /topics:
+    get:
+      description: Lists all known `Topics` of this Nakadi cluster
+      tags:
+        - monitoring
+      responses:
+        '200':
+          description: The request was successful. Returns a list `Topic`s in the response body.
+          schema:
+            type: array
+            description: An array of topics
+            items:
+              $ref: '#/definitions/Topic'
+        '401':
+          description: User is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+  '/topics/{topic}/events':
+    get:
+      tags:
+        - low-level-api
+      summary: |
+        Starts a stream delivery for the specified partitions of the given topic.
```

Comment:
I don't understand this resource method. How is the "specified partition" info passed? 
Via the x-nakadi-cursor header? If yes, the header need to be mandatory, not optional.

What is the difference to the GET /topics/{topic}/partitions/{partition} resource method?
Do we really need both LL methods?